frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -846,10 +846,48 @@ export function ContainersSection(props: { resource: KubeObjectInterface | null
     return statuses;
   }
 
+  function getEphemeralContainerStatuses() {
```

Comment:
This function should be added to the commit that introduces the ephemeral container statuses. Otherwise that commit will introduce changes that are not correct, only to have them corrected in another commit of the same PR.

Reply:
done!